integration.schema.json

```diff
@@ -19,7 +19,6 @@
     }
   },
   "required": [
-    "Request",
```

Comment:
Maybe keep the Request required (as we might extend this property later on) and only make the AuthenticationMethods within the Request object optional.

Reply:
Then we are left with just an empty node, that would look really strange. I'll keep it as is.